src/System.IO.Compression.Brotli/tests/System.IO.Compression.Brotli.Tests.csproj

```diff
@@ -28,8 +28,9 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <SupplementalTestData Include="$(NuGetPackageRoot)system.io.compression.testdata\1.0.8\content\**\*.*">
-      <Link>%(RecursiveDir)%(Filename)%(Extension)</Link>
-    </SupplementalTestData>
+    <!-- On OSX the Unicode file's build action property is overwritten with Compile. -->
```

Comment:
I don't think this removal is working.  Consider instead to excludeassets on the package and then manually add the content items.  I'm trying to reproduce this on my macbook to see what's going on.